api_inference/inference-api.ipynb

```diff
@@ -17,146 +17,137 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install openai"
+    "!pip install huggingface_hub"
```

Comment:
```suggestion
    "!pip install -q huggingface_hub"
```

This is totally upto you, I like the installation to be quite.

Reply:
I don't have a strong opinion on this and since it's a notebook, let's add the quiet flag :)